potofolio.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Document</title>
+</head>
+<body>
+    <div id="header">
+        <div id="logoplaceholder"></div>
+        <button class="menu">
+            <span class="line"></span>
+            <span class="line"></span>
+            <span class="line"></span>
+         </button>
```

Comment:
- [ ] For this section, the container should be a `header` tag.
-Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.